src/__experimental__/components/radio-button/radio-button.component.js

```diff
@@ -19,7 +19,6 @@ const RadioButton = ({
     ...props,
     onChange: handleChange,
     onBlur,
-    helpTabIndex: 0,
```

Comment:
What impact does this change have?

Reply:
This caused validation icon to be focused twice while the icon was displayed directly on radio component so I removed it. I've tested keyboard navigations on RadioButtons in different configurations and it seems to have no impact apart from fixing the bug that I've mentioned.